tests/release/e2e-test-push-image-to-pyxis.go

```diff
@@ -0,0 +1,204 @@
+
+		Expect(kubeController.CreateOrUpdateSigningSecret(
+			publicKey, publicSecretNameAuth, managedNamespace)).To(Succeed())
+
+		defaultEcPolicy := ecp.EnterpriseContractPolicySpec{
+			Description: "Red Hat's enterprise requirements",
+			PublicKey:   string(publicKey),
+			Sources: []ecp.Source{
+				{
+					Name: "ec-policies",
+					Policy: []string{
+						"git::https://github.com/hacbs-contract/ec-policies.git//policy/lib",
+						"git::https://github.com/hacbs-contract/ec-policies.git//policy/release",
+					},
+					Data: []string{
+						"git::https://github.com/hacbs-contract/ec-policies.git//data",
+					},
+				},
+			},
+			Configuration: &ecp.EnterpriseContractPolicyConfiguration{
+				Collections: []string{"minimal"},
+				Exclude:     []string{"cve"},
+			},
+		}
+
+		_, err = framework.AsKubeAdmin.CommonController.CreateServiceAccount(releaseStrategyServiceAccountDefault, managedNamespace, managednamespaceSecret)
+		Expect(err).NotTo(HaveOccurred())
+
+		err = framework.AsKubeAdmin.CommonController.LinkSecretToServiceAccount(managedNamespace, redhatAppstudioUserSecret, releaseStrategyServiceAccountDefault)
+		Expect(err).ToNot(HaveOccurred())
+
+		_, err = framework.AsKubeAdmin.ReleaseController.CreateReleasePlan(sourceReleasePlanName, devNamespace, applicationNameDefault, managedNamespace, "")
+		Expect(err).NotTo(HaveOccurred())
+
+		_, err = framework.AsKubeAdmin.ReleaseController.CreateReleaseStrategy("mvp-push-to-external-registry-strategy", managedNamespace, "push-to-external-registry", "quay.io/hacbs-release/pipeline-push-to-external-registry:main", releaseStrategyPolicyDefault, releaseStrategyServiceAccountDefault, paramsReleaseStrategyMvpPyxis)
+		Expect(err).NotTo(HaveOccurred())
+
+		_, err = framework.AsKubeAdmin.ReleaseController.CreateReleasePlanAdmission(targetReleasePlanAdmissionName, devNamespace, applicationNameDefault, managedNamespace, "", "", "mvp-push-to-external-registry-strategy")
+		Expect(err).NotTo(HaveOccurred())
+
+		_, err = framework.AsKubeAdmin.TektonController.CreateEnterpriseContractPolicy(releaseStrategyPolicyDefault, managedNamespace, defaultEcPolicy)
+		Expect(err).NotTo(HaveOccurred())
+
+		_, err = framework.AsKubeAdmin.TektonController.CreatePVCInAccessMode(releasePvcName, managedNamespace, corev1.ReadWriteOnce)
+		Expect(err).NotTo(HaveOccurred())
+
+		_, err = framework.AsKubeAdmin.HasController.CreateHasApplication(applicationNameDefault, devNamespace)
+		Expect(err).NotTo(HaveOccurred())
+
+		_, err = framework.AsKubeAdmin.HasController.CreateComponent(applicationNameDefault, componentName, devNamespace, gitSourceComponentUrl, "", containerImageUrl, "", "", true)
+		Expect(err).NotTo(HaveOccurred())
+
+	})
+
+	AfterAll(func() {
+
+		if !CurrentSpecReport().Failed() {
+			Expect(framework.AsKubeAdmin.CommonController.DeleteNamespace(devNamespace)).NotTo(HaveOccurred())
+			Expect(framework.AsKubeAdmin.CommonController.DeleteNamespace(managedNamespace)).NotTo(HaveOccurred())
```

Comment:
please add a [cleanup of user signup](https://github.com/redhat-appstudio/e2e-tests/blob/a607488b0989dccbc37f7a67a3a1d1f4a29f0164/tests/o11y/o11y.go#L36)

Reply:
done,